### DISCUSSION TITLE
Serenity theme: material icon font no longer showing up in p:menuitem

### DISCUSSION THREAD

**Discussion by MP2E** [primefaces/community#2608] [PrimeFaces Templates]
Example of one of the menu items XHTML:
<p:menuitem id="sr_lnk103" value="Services" icon="" outcome="/content/services.xhtml" />

When the left nav is minimized, the icon is supposed to appear but doesn't. When the left nav is not minimized, the menus look correct except the drop down materials icon is not showing for submenus.

Serenity version 6
Primefaces 13.0.3
Faces 4.0.1

**Comment by melloware** [comment]
@MP2E if you send me a DM on the Discord channel i may be able to help debug and fix it.  I have fixed some other premium template issues for others.

**Comment by tandraschko** [comment]
we deciced to only maintain:

          - "apollo"
          - "atlantis"
          - "avalon"
          - "diamond"
          - "freya"
          - "poseidon"
          - "ultima"
          - "verona"
          
for now, so i will close this issue for now
